src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -125,6 +135,21 @@ public boolean isUserRegistered(String username) {
    */
   public void setUsers(List<User> users) {
     this.users = users;
```

Comment:
Since you now have the loop that adds each user from `users`, you don't want to also do this first line, which overwrites the existing list of users (and also your initial Admin).  So drop this `this.users = users` line.

Reply:
You are right, I overlooked this. Fixed now.